allennlp/service/predictors/predictor.py

```diff
@@ -0,0 +1,134 @@
+import json
+import os
+from typing import Dict, Any, Optional, List
+
+from allennlp.common import Params, Registrable
+from allennlp.common.params import replace_none
+from allennlp.data.tokenizers import Tokenizer, WordTokenizer
+from allennlp.data.token_indexers import TokenIndexer
+from allennlp.data.dataset_readers import DatasetReader
+from allennlp.data import Vocabulary
+from allennlp.models import Model
+
+import numpy as np
+import torch
+
+JsonDict = Dict[str, Any]  # pylint: disable=invalid-name
+
+
+def sanitize(x: Any) -> Any:  # pylint: disable=invalid-name
```

Comment:
Can we move this function and the typedef above into `common/utils.py`, or somewhere similar?